about.html

```diff
@@ -0,0 +1,201 @@
+            <li><a href="./index.html"> Home</a></li>
+            <li>
+              <a id="logoiconabout" href="#logo-detail-section"> Logo Detail</a>
+            </li>
+            <li>
+              <a id="pastbatchiconabout" href="#past-Batch-section">
+                Past Batch Info</a
+              >
+            </li>
+            <li>
+              <a id="partnericonabout" href="#partner-section">Our Partners</a>
+            </li>
+            <li><a href="./index.html">Latest</a></li>
+          </ul>
+        </div>
+      </div>
+      
+      <div class="container about-home-container">
+       
+        <div class="row about-heading-2">
+          <h2>
+            AFRICA CULTURAL DANCE CONFERENCE<br>2023
+          </h2>
+        </div>
+        <div class="row about-para-heading">
+          <p>
+            The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+            So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together
+          </p>
+
+          <h3>
+            Please contact us per Email for nay further questions about the African Cultural Dace Conference 2023!
+          </h3>
+          <h3><a href="mailto:info@africanculturalconference.org"></a>info@africanculturalconference.org</a></h3>
+         
+        </div>
+        
+      </div>
+    </div>
+    <div class="about-para-heading">
+      <img
+      class="footer-img"
+      src="./images/logo.svg"          alt="educationlogo1"
+    />
+    </div>
+
+    <div class="row about-para-heading">
+      
+
+      <h1>
+        See the past conferences
+      </h1>
+      <div class="past-pic">
+        <div class="pastimage1">
+          <h2>2021</h2>
+          <p>Award Cermony in California</p>
+        </div>
+        <div class="pastimage2">
+          <h2>2022</h2>
+          <p>Award Cermony in America</p>
+        </div>
+      </div>
+    </div>
+    <!-- Partner section start from here -->
+    <div id="partner-section" class="container-fluid partner-section">
+      <div class="container partner-program-container">
+        <div class="row">
+          <div class="col-12 d-flex justify-content-center partner-main-text">
+            Partners
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center">
+          <div class="col-1 partner-main-text-line">line</div>
+        </div>
+      </div>
+
```

Comment:
- [ ] [REQUIRED] Forgot to create this section, please create it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235015013-22ceb8c9-c737-4911-a600-183acc61a894.png)

Reply:
- [ ] Please consider adding this section to the about page as requested by the previous reviewer
> * [ ]  [REQUIRED] Forgot to create this section, please create it 🙏🏼
> 
> ![image](https://user-images.githubusercontent.com/63915024/235015013-22ceb8c9-c737-4911-a600-183acc61a894.png)